### PR TITLE
#8022 - Add explanation for test file location

### LINKED ISSUE #8022: Explain where test file should be stored on 'An introduction to widget testing' page
### Page URL

https://docs.flutter.dev/cookbook/testing/widget/introduction/

### Page source

https://github.com/flutter/website/tree/main/src/cookbook/testing/widget/introduction.md

### Describe the problem

Should explain where the test file should be created as https://docs.flutter.dev/cookbook/testing/unit/introduction#2-create-a-test-file does

### Expected fix

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by JaredEzz**
_Description of what this PR is changing or adding, and why:_ The introduction for widget testing should explain where the test file should be created as the introduction for unit testing does. See https://docs.flutter.dev/cookbook/testing/unit/introduction#2-create-a-test-file 

_Issues fixed by this PR (if any):_ Fixes #8022 

## Presubmit checklist
- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

File: src/cookbook/testing/widget/introduction.md [hunk 1]
```diff
@@ -122,7 +145,7 @@ The `pumpWidget` method builds and renders the provided widget.
 Create a `MyWidget` instance that displays "T" as the title
 and "M" as the message.
 
-<?code-excerpt "test/main_step4_test.dart (main)"?>
+<?code-excerpt "test/my_widget_test.dart (main)"?>
```
Comment (domesticmouse) [inline src/cookbook/testing/widget/introduction.md:148]: I'm curious why this change was made

Comment (JaredEzz) [reply src/cookbook/testing/widget/introduction.md:148]: I'm not entirely sure how these file names are used, but if they're visible to the reader, I wanted it to be clear where the code in the excerpt would be located

**Review by sfshaza2 — COMMENTED** [review]
Hi @JaredEzz! We are freezing the website by end of day today. I can't land this PR because the build is failing. So, for now, I'm going to close it. Please feel free to re-open it once the website is unfrozen, but you must fix the build before it can land. Thx!
